helix-term/src/commands.rs

```diff
@@ -1551,7 +1574,13 @@ pub fn goto_reference(cx: &mut Context) {
 
     let offset_encoding = language_server.offset_encoding();
 
-    let pos = pos_to_lsp_pos(doc.text(), doc.selection(view.id).cursor(), offset_encoding);
+    let pos = if let Some(pos) =
+        pos_to_lsp_pos(doc.text(), doc.selection(view.id).cursor(), offset_encoding)
+    {
+        pos
+    } else {
+        return;
+    };
```

Comment:
I think you can use `match` to make it shorter here.

Reply:
Done.